libs/ui/molecules/src/lib/AvatarDropdown/__tests__/AvatarDropdown.spec.tsx

```diff
@@ -0,0 +1,35 @@
+import { render } from '@inheartive/ui/testing';
+import { AvatarDropdown } from '../AvatarDropdown';
+import { NativeRouter } from 'react-router-native';
+
+const dropdownList = [
```

Comment:
Maybe it would be better to move this drop-down list to a separate folder in libs / data / src? And use this in both files AvatarDropdown.stories.tsx and AvatarDropdown.spec.tsx